tests/manage/pv_services/pvc_snapshot/test_rbd_block_pvc_snapshot.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+from tests.helpers import wait_for_resource_state, create_pods
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.6')
+@pytest.mark.polarion_id('OCS-2361')
+class TestRbdBlockPvcSnapshot(ManageTest):
+    """
+    Tests to take snapshots of RBD Block VolumeMode PVCs
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(
+        self, project_factory, snapshot_restore_factory,
+        multi_pvc_factory, pod_factory
+    ):
+        """
+        Create PVCs and pods
+
+        """
+        self.pvc_size = 5
+
+        self.pvc_objs = multi_pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            size=self.pvc_size,
+            access_modes=[
+                f'{constants.ACCESS_MODE_RWX}-Block',
+                f'{constants.ACCESS_MODE_RWO}-Block'
+            ],
+            status=constants.STATUS_BOUND,
+            num_of_pvc=2,
+            wait_each=False
+        )
+
+        self.pod_objs = create_pods(
+            self.pvc_objs, pod_factory, constants.CEPHBLOCKPOOL,
+            pods_for_rwx=1, status=constants.STATUS_RUNNING
+        )
+
+    def md5sum(self, pod_obj):
+        """
+        Find md5sum
+
+        Args:
+            pod_obj (Pod): Instance of class Pod
+
+        Returns:
+            str: md5sum value
+
+        """
+        file_path = pod_obj.get_storage_path(storage_type='block')
+        md5sum_cmd_out = pod_obj.exec_cmd_on_pod(
+            command=f"bash -c \"md5sum {file_path}\"", out_yaml_format=False
+        )
```

Comment:
This will not work since md5sum not working on directory, it work only on files.

Reply:
We will get ```/dev/rbdblock``` as the value of  file_path if storage_type='block'